Makefile

```diff
@@ -0,0 +1,20 @@
+IMAGE_REPO="docker.io"
+NAME_SPACE="ibmcom"
+CONSOLE_IMAGE_VERSION=1.3.0
+
+CONSOLE_NAME=ibm-storage-odf-plugin
+
+CONSOLE_IMAGE=$(IMAGE_REPO)/${NAME_SPACE}/$(CONSOLE_NAME)
```

Comment:
Can we add the `CONSOLE_IMAGE` in here, instead of adding it to every docker command?

Reply:
fixed